### PR TITLE
POK-35 angular routing main menu

### PR DESCRIPTION
**Pull request by AmazingHooman**

### CODE REVIEW

**Comment by maeggimaesser** [comment]
denke ist gut

File: PokeOneWeb/Frontend/pokeoneweb/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -4,6 +4,10 @@ import { HomeComponent } from './pages/home/home.component';
 import { LoadingComponent } from './pages/loading/loading.component';
 import { NotFoundComponent } from './pages/not-found/not-found.component';
 import { DynamicRouteGuard } from './core/guards/dynamic-route.guard';
+import { FaqComponent } from './pages/faq/faq.component';
+import { AboutComponent } from './pages/about/about.component';
+import { GuidesComponent } from './pages/guides/guides.component';
+import { LocationsComponent } from './pages/locations/locations.component';
```
Comment (Finrod-Amandil) [inline PokeOneWeb/Frontend/pokeoneweb/src/app/app-routing.module.ts]: Please follow the naming convention of the other pages and name the component LocationListComponent (location-list.component)

File: PokeOneWeb/Frontend/pokeoneweb/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -66,6 +70,38 @@ const routes: Routes = [
                 (m) => m.ItemListModule
             ),
     },
+    {
+        path: 'g',
+        component: GuidesComponent,
```
Comment (Finrod-Amandil) [inline PokeOneWeb/Frontend/pokeoneweb/src/app/app-routing.module.ts]: Put new pages into own modules (lazy loading, components are only loaded when required)

Compare other pages or use Angular CLI: `ng g c folder/component-name --module=module-name`

File: PokeOneWeb/Frontend/pokeoneweb/src/app/core/components/navbar/navbar.component.html [hunk 3]
```diff
@@ -2,32 +2,38 @@
...
+            menuItemLink="/i"
             menuItemImage="./assets/img/navbar/items.png">
         </pokeone-navbar-item>
 
         <pokeone-navbar-item 
-            menuItemCaption="Moves"
-            menuItemImage="./assets/img/navbar/moves.png">
+            menuItemCaption="Guides"
+            menuItemLink="/g"
+            menuItemImage="./assets/img/navbar/guides.png">
```
Comment (Finrod-Amandil) [inline PokeOneWeb/Frontend/pokeoneweb/src/app/core/components/navbar/navbar.component.html:25]: Image is blurry. Use a pixel art scaling tool to scale the image up to 120x120px, i.e. https://lospec.com/pixel-art-scaler/

**Review by Finrod-Amandil — CHANGES_REQUESTED** [review]

File: PokeOneWeb/Frontend/pokeoneweb/src/app/app-routing.module.ts (hunk 1 above)
Comment (AmazingHooman) [reply PokeOneWeb/Frontend/pokeoneweb/src/app/app-routing.module.ts]: Done

File: PokeOneWeb/Frontend/pokeoneweb/src/app/app-routing.module.ts (hunk 2 above)
Comment (AmazingHooman) [reply PokeOneWeb/Frontend/pokeoneweb/src/app/app-routing.module.ts]: Done

File: PokeOneWeb/Frontend/pokeoneweb/src/app/core/components/navbar/navbar.component.html (hunk 3 above)
Comment (AmazingHooman) [reply PokeOneWeb/Frontend/pokeoneweb/src/app/core/components/navbar/navbar.component.html:25]: Done

File: PokeOneWeb/Frontend/pokeoneweb/src/app/pages/about/about.module.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import { FormsModule } from '@angular/forms';
+import { RouterModule } from '@angular/router';
+
+@NgModule({
+  declarations: [AboutComponent],
+  imports: [
+    CommonModule,
+    AboutRoutingModule,
+    ComponentsModule,
+    NgSelectModule,
```
Comment (Finrod-Amandil) [inline PokeOneWeb/Frontend/pokeoneweb/src/app/pages/about/about.module.ts]: Bitte NgSelectModule und VirtualScrollerModule bei den page stubs rausnehmen, braucht es möglicherweise nicht.

**Review by Finrod-Amandil — CHANGES_REQUESTED** [review]

File: PokeOneWeb/Frontend/pokeoneweb/src/app/pages/about/about.module.ts (hunk 4 above)
Comment (AmazingHooman) [reply PokeOneWeb/Frontend/pokeoneweb/src/app/pages/about/about.module.ts]: Habe ich bei About, Guides und Faq entfernt

**Review by Finrod-Amandil — APPROVED** [review]

**Pull request merged**